util/helpers.ts

```diff
@@ -38,6 +38,18 @@ export async function deployFeeManager(deployer: Signer, owner: string, treasury
   return feeManager;
 }
 
+export async function deployPayoutManager(deployer: Signer, confirmations: number = config.confirmationsForDeploy): Promise<Contract> {
+  const MerkleTreePathValidator = await ethers.getContractFactory("MerkleTreePathValidator", deployer);
```

Comment:
We should deploy this separately so that we can use the same contract in the future if we need it.

Reply:
well noted, I'll separate the process in the QueryService PR